frontend/packages/dev-console/src/components/helm/HelmReleaseUpgradePage.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
```

Comment:
Rename this one for consistency. `HelmUpgradePage`.

Reply:
removed this component